application/templates/cms/edit_measure_page.html

```diff
@@ -286,7 +292,9 @@ <h3 class="heading-medium">Data</h3>
                                     </td>
                                     <td>
                                         {% if 'UPDATE' in available_actions %}
-                                            <a href="{{ url_for('cms.delete_upload', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version, upload_guid=upload.guid ) }}">delete</a>
+                                            <form action="{{ url_for('cms.delete_upload', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version, upload_guid=upload.guid ) }}" method="post">
```

Comment:
I think this should need a csrf token in the form to work?  At least, it should if we have csrf protection turned on for all POST requests?

Reply:
I thought that, but it works without. How do you require it?

Reply:
It's a Flask configuration thing I think.  

Need to add it in `create_app`
http://flask-wtf.readthedocs.io/en/stable/csrf.html

Reply:
@TheDoubleK I can't quite figure out whereabout this should go in our application?

Reply:
Sorry, was at the dentist with child so terse reply before.

I'll have a go at getting it working now.

Reply:
👍